document-sync-job/base/cronjob.yaml

```diff
@@ -0,0 +1,101 @@
+---
+kind: CronJob
+apiVersion: batch/v1beta1
+metadata:
+  name: document-sync
+spec:
+  schedule: "@daily"
+  suspend: true
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 2
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - image: quay.io/thoth-station/document-sync-job:v0.1.0
+              name: document-sync-job
+              env:
+                - name: THOTH_DOCUMENT_SYNC_DST
+                  value: "s3://thoth-store/data/aws-prod"
```

Comment:
Is this the destination bucket where prod has thoth data?

Reply:
yes, that is correct.
`thoth` was already taken, so we had to use the `thoth-store`.